Doppler.ContactPolicies.Business.Logic/Services/IContactPoliciesService.cs

```diff
@@ -5,8 +5,8 @@ namespace Doppler.ContactPolicies.Business.Logic.Services
 {
     public interface IContactPoliciesService
     {
-        Task<ContactPoliciesSettingsDto> GetContactPoliciesSettingsAsync(string accountName);
-        Task UpdateContactPoliciesSettingsAsync(string accountName, ContactPoliciesSettingsDto contactPoliciesSettings);
+        Task<ContactPoliciesSettingsDto> GetContactPoliciesSettingsAsync(int idUser);
```

Comment:
I know that this is not in PR´s scope but what do you think about that?
```suggestion
        Task<ContactPoliciesSettingsDto> GetContactPoliciesSettingsByIdUserAsync(int idUser);
```

Reply:
👍